tests/e2e/pkg/tester/skip_regex.go

```diff
@@ -34,6 +34,12 @@ func (t *Tester) setSkipRegexFlag() error {
 		return nil
 	}
 
+	kopsVersion, err := t.getKopsVersion()
+	if err != nil {
+		return err
+	}
+	isPre28 := kopsVersion < "1.28"
```

Comment:
nit: This will evaluate to false for any future 1.28 pre-release builds

Reply:
And we can't inspect the Cluster manifest because `spec.networking.cilium.version` isn't guaranteed to be set. I suppose one alternative would be to inspect the daemonset itself with the kubectl binary that the tester downloads: https://github.com/kubernetes/kops/blob/7e06e4c4162cd7d5229c227f89815977934f8fc4/tests/e2e/pkg/tester/kubectl.go#L132-L134

Reply:
Evaluating false for future 1.28 pre release builds is desired behavior